packages/edit-site/src/components/global-styles/shadows-edit-panel.js

```diff
@@ -394,8 +391,7 @@ function ShadowItem( { shadow, onChange, canRemove, onRemove } ) {
 						{ canRemove && (
 							<FlexItem>
 								<Button
-									// TODO: Switch to `true` (40px size) if possible
-									__next40pxDefaultSize={ false }
+									__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> styles --> shadows --> add shadow. Open shadow and check for `drop shadow` remove icon button.

**Note:** It already has a 40px size as height is auto.

| Before | After     |
| :-------- | :------- |
| <img width="625" alt="drop-shadow-remove-before" src="https://github.com/user-attachments/assets/b009145a-e40a-46cc-84d3-72566557bb69"> | <img width="625" alt="drop-shadow-remove-after" src="https://github.com/user-attachments/assets/0515cf38-ab54-419e-b98d-609cbac83a74"> |